packages/iov-ethereum/src/utils.ts

```diff
@@ -56,6 +56,15 @@ export function normalizeHex(input: string): string {
   }
 }
 
+/**
+ * Takes a hex representation optionally prefixed with 0x and returns an Ethereum-friendly
+ * representation with a prefix.
+ */
+export function toEthereumHex(input: string): string {
```

Comment:
I thought this could take Uint8Array directly to avoid unnecessary string operations and shorter caller code in most places.